common/azure-common/src/test/java/com/azure/common/implementation/util/FluxUtilTests.java

```diff
@@ -29,12 +32,22 @@
 
 public class FluxUtilTests {
 
+    @BeforeClass
+    public static void setUp() throws Exception {
+        File dir = new File("target");
+        if (!dir.exists()) {
+            Files.createDirectory(Paths.get("target"));
+        }
+    }
```

Comment:
` setUp()` can be removed, please see comment for `createFileIfNotExist` helper.

Reply:
Done